api/src/app/modules/project/project.dto.ts

```diff
@@ -147,6 +152,9 @@ export class ProjectResponse {
 
   @ApiProperty({ required: true })
   operationEndYear: number;
+
```

Comment:
Shouldn't the bcts name also exist on the create request?

Reply:
Hi Basil, correct, the name does exist on the create request on line 59 (`ProjectCreateRequest`).
The line 157 is for the `ProjectResponse`.